keras_core/backend/jax/distribution_lib.py

```diff
@@ -0,0 +1,56 @@
+"""!!!DO NOT USE!!!
+
+Distribution related class for JAX backend.
+
+This is just a prototype and we might want to unify it
+with other backends in the future.
+"""
+import jax
+
+
+def list_devices(device_type=None):
+    """Return all the available devices based on the device type.
+
+    Note that this should return the global devices in a distributed setting.
+
+    Args:
+        device_type: string of 'CPU', 'GPU' or 'TPU'. Default to GPU or TPU if
+            available when device_type is not provided. Otherwise will return
+            the CPU devices.
```

Comment:
QQ: is defaulting to CPU consistent with what happens with JAX and TF list_device?

Reply:
Yes, the priority order of device type is "GPU" == "TPU" > "CPU", when the device type is not specified. 

This is same between JAX and tf.dtensor.

Reply:
I see your question, I was incorrectly converting None to "CPU", and it has been changed back now.

Reply:
Btw, I don't think we have GPU/TPU test setup for OSS build. Depends on how hard to hookup the TPU/GPU with github actions, we might also need to consider setting the device test internally.